test/expected/resolve___schema.out

```diff
@@ -237,10 +245,6 @@ begin;
                      "kind": "SCALAR",               +
                      "name": "JSON"                  +
                  },                                  +
-                 {                                   +
-                     "kind": "INPUT_OBJECT",         +
-                     "name": "JSONFilter"            +
-                 },                                  +
```

Comment:
Removing this type is a user facing breaking change. However, using `JSONFilter` resulted in an error 100% of the time, so no messaging is required around its removal.